gems/csv/3.3/manifest.yaml

```diff
@@ -1,3 +1,4 @@
 dependencies:
   - name: forwardable
   - name: stringio
+  - name: tmpdir
```

Comment:
It does not appear to be necessary.

Reply:
A metadata.yaml is required for types used only for testing.

Here is a good examples.
https://github.com/ruby/gem_rbs_collection/blob/main/gems/redis/4.2/_test/metadata.yaml

Reply:
Yeah, I misunderstood that `bin/test` tests Ruby code behaviors. If so, we have needed `tmpdir` to generate files for testing. But now I understood it's not required.